daily-shoutout/app.py

```diff
@@ -55,11 +56,22 @@ def sendTweet(tweet_string):
     client.create_tweet(text = tweet_string)
 
 
+def returnResponse(response_string):
+    return{
+        "statusCode": 200,
+        "body": json.dumps(response_string)
+    }
+
+
 def lambda_handler(event, context):
     person = people[get_random_index(len(people))]
     cause = causes[get_random_index(len(causes))]
+    message_string = f"Noted billionare {person} once again contributes nothing to {cause}."
```

Comment:
Would also be cool to extract this part out into a separate function to make it a little easier to start returning different kinds of text.

Reply:
we can do that when it needs to be done. :-)